ui/components/app/snaps/snap-authorship-header/snap-authorship-header.js

```diff
@@ -28,6 +28,12 @@ const SnapAuthorshipHeader = ({
   className,
   boxShadow = 'var(--shadow-size-lg) var(--color-shadow-default)',
 }) => {
+  console.log(
```

Comment:
despite the props being passed in correctly the first time, there is something that is overriding this call and passing in undefined for everything making. I can see this in the logs.

```
2: snap-authorship-header.js:31 SNAPS/ snap-authorship-header.js called with npm:@metamask/snap-simple-keyring-snap undefined var(--shadow-size-lg) var(--color-shadow-default)

snap-authorship-header.js:31 SNAPS/ snap-authorship-header.js called with undefined undefined var(--shadow-size-lg) var(--color-shadow-default)
```